src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/attn_quant.cpp

```diff
@@ -0,0 +1,248 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <float.h>
+
+#include <cmath>
+#include <cstring>
+#include <iostream>
+#include <limits>
+#include <type_traits>
+
+#if defined(HAVE_AVX2) || defined(HAVE_AVX512F)
+#    include <immintrin.h>
+#endif
+
+#include "openvino/core/type/bfloat16.hpp"
+#include "openvino/core/parallel.hpp"
+#include "common.hpp"
+#include "attn_quant.hpp"
+
+namespace ov {
+namespace Extensions {
+namespace Cpu {
+namespace XARCH {
+
+using namespace ov;
+
+template<typename T>
+static void quant_u8(uint8_t* a, T* b, size_t n, float& scale, float& zp) {
```

Comment:
minor: common practice is to put src memory as first arg. Also please give `a` and `b` args more meaningful name and don't forget about `const` attr.

Reply:
Done.